GUI/MainWindow.xaml.cs

```diff
@@ -383,7 +383,9 @@ private void AddAFile(string draggedFilePath)
             // we need to get the filename before parsing out the extension because if we assume that everything after the dot
             // is the extension and there are dots in the file path (i.e. in a folder name), this will mess up
             var filename = Path.GetFileName(draggedFilePath);
-            var theExtension = filename.Substring(filename.IndexOf(".")).ToLowerInvariant();
+            var theExtension = Path.GetExtension(filename).ToLowerInvariant();
```

Comment:
this looks fine, you will also need to change code in MetaMorpheus/TaskLayer/GPTMDTask/GPTMDTask.cs line ~201 for GPTMD database writing to make sure the correct file extension (.xml) is written there and not something weird like .xml.xml when searching with an .xml.gz